web/routing.go

```diff
@@ -57,6 +57,10 @@ func SetupAppsHandler(appsHandler echo.HandlerFunc) echo.HandlerFunc {
 	mws := []echo.MiddlewareFunc{
 		middlewares.LoadAppSession,
 		middlewares.CheckIE,
+		middlewares.Accept(middlewares.AcceptOptions{
```

Comment:
Wouldn't it be better to put this middleware before `CheckIE`, like in `SetupRoutes`?